Successfully updated:
Created Frame node m8vCSk0Ym
Created Frame node J5TyURRPc
Created Text node TsbbNsNhW
Created Text node O7En4lLeH
Created Frame node tZIvwUEtu
Updated attributes for node m8vCSk0Ym
Updated attributes for node J5TyURRPc
Updated text for node TsbbNsNhW
Updated attributes for node TsbbNsNhW
Updated text for node O7En4lLeH
Updated attributes for node O7En4lLeH
Updated attributes for node tZIvwUEtu
Reordered node J5TyURRPc within parent m8vCSk0Ym to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3881,20 +3881,73 @@
       width="fit-content"
       height="fit-content"
       top="0px"
       left="0px"
       font="Inter"
       textTruncation="2"
   >
     Truncated Text that is long enough to be truncated
   </TruncatedTextThatIsLongEnoughToBeTruncated>
 </Frame>
 <Frame
     nodeId="yKtdVfGM5"
     position="absolute"
     width="100px"
     height="100px"
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
     backgroundColor="rgb(200, 200, 200)"
  />
+<Stack
+    nodeId="m8vCSk0Ym"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="J5TyURRPc"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="TsbbNsNhW"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="O7En4lLeH"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="tZIvwUEtu"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.